### PR TITLE
Support piping into kubesec with `-` and `/dev/stdin`

### LINKED ISSUE #104: Support `-` in place of `/dev/stdin` and/or fall back to stdin if no file provided
`cat file1.yml file2.yml | kubesec scan -` and `cat file1.yml file2.yml | kubesec scan` don't work

Should behave like `cat file1.yml file2.yml | kubesec scan /dev/stdin`

### PR DESCRIPTION
**Pull request by 06kellyjac**
Resolves #104

### CODE REVIEW

File: test/_helper.bash [hunk 1]
```diff
@@ -95,6 +95,15 @@ else
     ./../${BIN_UNDER_TEST:-false} "${ARGS}";
   }
 
+  _app_pipe() {
+     local ARGS="${@:-}"
+    if [[ "${BIN_UNDER_TEST}" != "" ]]; then
+      # remove --json flags
+      ARGS=$(echo "${ARGS}" | sed -E 's,--json,,g')
+    fi
+    ./../${BIN_UNDER_TEST:-false} - <"${ARGS}";
```
Comment (sublimino) [inline test/_helper.bash:104]: Minor nit: this isn't a pipe : ] Might want to rename to `_app_redirect`, change to pipe, or test both (I don't believe there's any difference when redirecting STDIN but I'm not sure)

Also extra prefixed space on line 99

**Comment by 06kellyjac** [comment]
I need to add a test for an empty `kubesec scan` and fix it's behavior, it's currently erroring in a non-clean way
